shared/chat/conversation/info-panel/index.desktop.js

```diff
@@ -23,4 +23,19 @@ const InfoPanel = (props: Props) => (
   </Box>
 )
 
-export default InfoPanel
+const ChannelInfoPanel = (props: ChannelInfoPanelProps) => (
```

Comment:
there is a generic scrollView in common-adapters that gives you a scrollable div / nativescrollview so you can likely merge this code also